src/Plugin/PhpUnit.php

```diff
@@ -219,11 +215,55 @@ protected function runConfig($directory, $configFile, $logFormat)
                     )
                 );
             }
+
+            return $this->checkRequiredCoverage($currentCoverage);
         }
 
         return $success;
     }
 
+    /**
+     * Extracts coverage from output
+     *
+     * @param string $output
+     *
+     * @return array
+     */
+    protected function extractCoverage($output)
+    {
+        preg_match(
+            '#Classes:[\s]*(.*?)%[^M]*?Methods:[\s]*(.*?)%[^L]*?Lines:[\s]*(.*?)\%#s',
+            $output,
+            $matches
+        );
+
+        return [
+            'classes' => !empty($matches[1]) ? $matches[1] : '0.00',
+            'methods' => !empty($matches[2]) ? $matches[2] : '0.00',
+            'lines'   => !empty($matches[3]) ? $matches[3] : '0.00',
+        ];
+    }
+
+    /**
+     * Checks required test coverage
+     *
+     * @param array $coverage
+     *
+     * @return bool
+     */
+    protected function checkRequiredCoverage($coverage)
+    {
+        foreach ($coverage as $key => $currentValue) {
+            if ($allowedValue = $this->options->getOption(implode('_', ['required', $key, 'coverage']))) {
+                if (floatval($currentValue) < floatval($allowedValue)) {
```

Comment:
@slavkluev You should use `bccomp` for comparison of floats. Could you improve it please?

Reply:
Sure. It's a good point.